### ISSUE TITLE
'add-podeauthaccess' is not recognized as a name of a cmdlet

### ISSUE THREAD

**Issue by goldpaint** [Badgerati/Pode#1199]
### Describe the Bug
Add-PodeAuthAccess is missing from module

### Steps To Reproduce
import-module pode
get-command add-podeauthaccess
Get-Command: The term 'add-podeauthaccess' is not recognized as a name of a cmdlet, function, script file, or executable program.
Check the spelling of the name, or if a path was included, verify that the path is correct and try again.

### Expected Behavior
Add-PodeAuthAccess should be present in Pode 2.9.0

### Platform
 - OS: Windows 10
 - Browser: n/a
 - Versions:
   - Pode: 2.9.0
   - PowerShell: 7.3.9

### Additional Context
I'm sure I must be missing something... I used Pode a few years ago but am struggling to set up authorization in IIS by following the new documentation as it looks like one or more functions are missing.
Pode was installed from PowerShell Gallery but I have also tried downloading the latest release from Github and importing the module directly (both show as version 2.9.0)

**Comment by Badgerati** [comment]
Hi @goldpaint,

The method you're after is `Add-PodeAccess`, where are you seeing it as `Add-PodeAuthAccess`? 🤔

**Comment by goldpaint** [comment]
Hi @Badgerati 

I now can't find where I got this from! Very confusing.
I did find a mention of it here - https://github.com/Badgerati/Pode/issues/992#issuecomment-1678004748 - but I guess it must have changed since.
Can only assume I stumbled upon some old documentation somewhere but can't for the life of me find it now.

Apologies for wasting your time and thanks for pointing me in the right direction - everything is working perfectly now using Add-PodeAccess
